src/tests/ftest/util/telemetry_utils.py

```diff
@@ -561,7 +561,7 @@ class TelemetryUtils():
         "engine_mem_vos_dtx_cmt_ent_48",
         "engine_mem_vos_vos_obj_360",
         "engine_mem_vos_vos_lru_size",
-        "engine_mem_dtx_dtx_leader_handle_336",
+        "engine_mem_dtx_dtx_leader_handle_352",
```

Comment:
What is "336" and "352"?

Reply:
oh, this is the dtx_leader_handler size in bytes, which is used in telemetry name here. Hmm, this test might needs to be smarter.